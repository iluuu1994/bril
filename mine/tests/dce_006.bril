@main {
  jmp one;
.two:
  print a;
  jmp three;
.one:
  a: int = const 1;
  b: int = const 2;
.three:
  # FIXME: Empty blocks are removed, even if they are a jump target.
  ret;
}
