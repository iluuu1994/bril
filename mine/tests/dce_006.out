@main {
  jmp one;
.two:
  print a;
  jmp three;
.one:
  a: int = const 1;
.three:
  ret;
}
